docs/src/components/Snippets/glossary_snippets.js

```diff
@@ -0,0 +1,202 @@
+import React from "react";
+
+export const Glossary = {
+  Tools: {
+    aztec: () => (
+      <p>
+        The CLI tool (with an extensive set of parameters) that enables users to
+        perform a wide range of tasks. It can: run a node, run a sandbox,
+        execute tests, generate contract interfaces for javascript...
+      </p>
+    ),
+    aztec_nargo: () => (
+      <p>
+        The command line tool used to compile Aztec contracts. It gets its name
+        from the Noir compiler,
+        <code>nargo</code>. <code>aztec-nargo</code> is really just a specific
+        version of <code>nargo</code>.
```

Comment:
```suggestion
        version of <code>nargo</code> and a transpiler for turning contract public function code from Noir brillig bytecode into Aztec Virtual Machine (AVM) bytecode.
```

Reply:
This definition was as simple as the previous one, and I agree it wasn't so complete. I'll consolidate this addition with less of the original text.